src/main.tsx

```diff
@@ -5,7 +5,7 @@ import { ImagePickerUtils } from '../lib/utils/imagePickerUtils.ts'
 
 const App = () => {
   const [content, setContent] = useState<string>(
-    '<p> ashdkasd </p> <img src = "https://picsum.photos/200/300" width="100%" height="auto" /> <p> hello </p>'
+    '<p> ashdkasd </p> <img src = "https://picsum.photos/200/300" width ="75" height="112" /> <p> hello </p>'
```

Comment:
??

Is this an app to test the Tapwrite package? Why is this being pushed to Github?